apps/laboratory/tests/smart-account.spec.ts

```diff
@@ -13,32 +13,59 @@ testModalSmartAccount.beforeEach(async ({ modalValidator }) => {
   await modalValidator.expectConnected()
 })
 
-testModalSmartAccount('it should sign with eoa', async ({ modalPage, modalValidator }) => {
-  await modalPage.sign()
-  await modalPage.approveSign()
-  await modalValidator.expectAcceptedSign()
+testModalSmartAccount('it should use a Smart Account', async ({ modalPage, modalValidator }) => {
+  const walletModalPage = modalPage as ModalWalletPage
```

Comment:
@tomiir This tests says `it should use a Smart Account` but it's not switching to SA, is it expected?

Reply:
I changed the fixture so that it switches to Polygon first, which defaults to the SA cause the EOA has no funds

Reply:
I see. I didn't know it's SA by default on Polygon. Btw, it's switching to Sepolia, rather than Polygon, it's SA by default also? https://github.com/WalletConnect/web3modal/pull/2529/files#diff-4fcba1bfcbc52350c9de92ef4be0a3fda9781493d37f43eb0a6c433294b411d5R26

Reply:
My bad I meant sepolia (same behavior)